server/routes/api.py

```diff
@@ -229,7 +230,12 @@ def specific_post(author_id: str, post_id: str) -> Response:
         db.session.add(post)
         db.session.commit()
         post.push()
-        return Response(status=httpStatus.CREATED)
+        # TODO: update documentation for this
+        return (
+            make_response(jsonify(post.json())),
+            httpStatus.OK,
```

Comment:
I think `201 Created` is more appropriate here. 

If you have a good reason to keep it at this could you please change the assertion on line 161 in `server/tests.py` to 200?

Reply:
to be honest, it's very unintelligent answer. I was unsure if we could return a body response with `201 Created` so I just went ahead with `200 OK`. I think I looked up and it mentioned something weird about having to do something to return a body response with `201`.

Reply:
Okay, if you want to keep 200 could you update the assertion in the tests for that endpoint?

Reply:
will do! how do you run the tests?

Reply:
I put it back to 201 :)